### ISSUE TITLE
Fix style_loss in Style_Transfer_Solution.ipynb

### ISSUE THREAD

**Issue by petercerno** [udacity/deep-learning-v2-pytorch#174]
Hi,

it is not necessary to divide the `layer_style_loss` by `(d * h * w)` since `layer_style_loss` is already computed as MSE over Gram Matrix elements, i.e. it is the average error *per element*.

```
# the style loss for one layer, weighted appropriately
layer_style_loss = style_weights[layer] * torch.mean((target_gram - style_gram)**2)
# add to the style loss
style_loss += layer_style_loss / (d * h * w)
```

Correct code:

```
style_loss += layer_style_loss
```

One can then set the `style_weight` to a much smaller number, e.g. `style_weight = 1e3` works great.

Cheers,

Peter

**Comment by puhach** [comment]
I believe it's needed since according to the paper the contribution of each layer to the total style loss is:
![0_nm5fyUiC0MK6adXL](https://user-images.githubusercontent.com/12899171/70372000-70a8af00-18e2-11ea-8af9-da4676cf0dd7.png)

In our case Nl is d and Ml = h * w. Due to `torch.mean()` we divide by d * h * w instead of d^2 * (h*w)^2 and 4 is a constant which, I guess, is omitted in favor of `style_weight` factor.

**Comment by abhiojha8** [comment]
Happy to review and approve a PR for this. Closing this for now as we will incorporate this feedback in future refreshes.
